sdk/identity/identity/src/credentials/visualStudioCodeCredential.ts

```diff
@@ -176,7 +176,7 @@ export class VisualStudioCodeCredential implements TokenCredential {
 
     // We want to make sure we use the one assigned by the user on the VSCode settings.
     // Or just `Azure` by default.
-    const cloudName = getPropertyFromVSCode("azure.cloud") || "Azure";
+    const cloudName = getPropertyFromVSCode("azure.cloud") || "AzureCloud";
```

Comment:
Is `AzureCloud` an arbitrary name we choose or is it meaningful to the auth process?

Reply:
@daviwil "AzureCloud" is the default cloud name set by the VSCode extension.

Reply:
Sounds good!